src-tauri/src/main.rs

```diff
@@ -41,7 +41,7 @@ fn main() {
                 println!("Using webui path: {}", var_path);
                 std::path::Path::new(var_path)
             } else {
-                panic!("AW_WEBUI_DIR environment variable not set!\nTry running make");
+                panic!("AW_WEBUI_DIR environment variable not set, This should be unreachable!");
```

Comment:
The code in main.rs should no longer be needed. If `AW_WEBUI_DIR` is given during the build, then it would be embedded in aw-server-rust at build-time, and we shouldn't override the asset path here.